packages/cli/package.json

```diff
@@ -45,7 +44,7 @@
   ],
   "dependencies": {
     "@axe-core/webdriverjs": "^4.7.3",
-    "axe-core": "^4.7.0",
+    "axe-core": "~4.7.0",
```

Comment:
Axe-core 4.8.1 was released and I don't want to take it with this conversion, so pinned all axe-core versions to 4.7